package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "citizen",
-  "version": "0.3.3",
+  "version": "1.0.0",
```

Comment:
I don't want to version up now.

Reply:
The reason version went up is that this MR introduces breaking changes in citizen cli